tfhe/src/high_level_api/integers/unsigned/scalar_ops.rs

```diff
@@ -728,6 +750,9 @@ generic_integer_impl_scalar_operation!(
         (super::FheUint128, u8, u16, u32, u64, u128),
         (super::FheUint160, u8, u16, u32, u64, u128, U256),
         (super::FheUint256, u8, u16, u32, u64, u128, U256),
+        (super::FheUint512, u8, u16, u32, u64, u128, U256, U512),
+        (super::FheUint1024, u8, u16, u32, u64, u128, U256, U512, U1024),
+        (super::FheUint2048, u8, u16, u32, u64, u128, U256, U2048),
```

Comment:
likely missing 512 and 1024 for all 2048 calls as it was done before the others were available ? same thing below for all macro calls